src/main/java/edu/kit/ipd/crowdcontrol/objectservice/Main.java

```diff
@@ -219,4 +218,42 @@ protected PasswordAuthentication getPasswordAuthentication() {
                 origin
         ).init();
     }
+
+    private static void check(Config config) throws ConfigException {
+        if (config == null || config.moneytransfer == null || config.database == null || config.deployment == null ||
+                config.platforms == null || config.database.writing == null || config.database.readonly == null ||
+                config.database.dialect == null || config.database.url == null || config.database.readonly.user == null ||
+                config.database.readonly.password == null || config.database.writing.user == null ||
+                config.database.writing.password == null || config.deployment.origin == null ||
+                config.moneytransfer.notificationMailAddress == null || config.moneytransfer.parsingPassword == null) {
+            throw new ConfigException("Configuration file incomplete");
```

Comment:
Lets see the following situation:
- someone creates a new config
- startup failed with "Configuration file incomplete"

Maybe we need a way to get here the name of the option which is missing.